src/mprisplayer.cpp

```diff
@@ -0,0 +1,585 @@
+// -*- c++ -*-
+
+/*!
+ *
+ * Copyright (C) 2015 Jolla Ltd.
+ *
+ * Contact: Valerio Valerio <valerio.valerio@jolla.com>
+ * Author: Andres Gomez <andres.gomez@jolla.com>
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+
+#include "mprisplayer.h"
+
+#include "mprisplayer_p.h"
+
+#include <qqmlinfo.h>
+
+#include <QDBusConnection>
+#include <QDBusMessage>
+#include <QDBusPendingCall>
+#include <QDBusPendingCallWatcher>
+#include <QDBusReply>
+
+static const QString serviceNamePrefix = QStringLiteral("org.mpris.MediaPlayer2.");
+static const QString mprisObjectPath = QStringLiteral("/org/mpris/MediaPlayer2");
+static const QString dBusPropertiesInterface = QStringLiteral("org.freedesktop.DBus.Properties");
+static const QString dBusPropertiesChangedSignal = QStringLiteral("PropertiesChanged");
+
+
+MprisPlayer::MprisPlayer(QObject *parent)
+    : QObject(parent)
+    , QDBusContext()
+    , m_mprisRootAdaptor(new MprisRootAdaptor(this))
+    , m_mprisPlayerAdaptor(new MprisPlayerAdaptor(this))
+    , m_serviceName()
+    , m_canQuit(false)
+    , m_canRaise(false)
+    , m_canSetFullscreen(false)
+    , m_desktopEntry()
+    , m_fullscreen(false)
+    , m_hasTrackList(false)
+    , m_identity()
+    , m_supportedUriSchemes()
+    , m_supportedMimeTypes()
+    , m_canControl(false)
+    , m_canGoNext(false)
+    , m_canGoPrevious(false)
+    , m_canPause(false)
+    , m_canPlay(false)
+    , m_canSeek(false)
+    , m_loopStatus(Mpris::None)
+    , m_maximumRate(1)
+    , m_metadata()
+    , m_typedMetadata()
+    , m_minimumRate(1)
+    , m_playbackStatus(Mpris::Stopped)
+    , m_position(0)
+    , m_rate(1)
+    , m_shuffle(false)
+    , m_volume(0)
```

Comment:
Same here, the ones with basic types must be initiated, the ones with default constructors(QStuff, QString, ...) no need to have here in the store if no default is needed.